deploy/charts/benthos-collector/templates/rbac.yaml

```
@@ -21,6 +21,32 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   name: {{ include "benthos-collector.fullname" . }}
 subjects:
+- kind: ServiceAccount
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)